mui/mui.py

```diff
@@ -45,6 +45,24 @@
 BinaryView.set_default_session_data("mui_state", None)
 BinaryView.set_default_session_data("mui_evm_source", None)
 
+def offset_address(bv: BinaryView, addr: int):
+    """Offsets addresses to take into consideration position independent executables (PIE)"""
+    BASE_DYN_ADDR = 0x555555554000
+    BASE_DYN_ADDR_32 = 0x56555000
```

Comment:
Please add a note that these values are copied from here https://github.com/trailofbits/manticore/blob/c3eabe03cf94f410bedd96d850df09cb0bda1711/manticore/platforms/linux.py#L954-L956